packages/ckeditor5-table/src/tableui.js

```diff
@@ -277,14 +277,15 @@ export default class TableUI extends Plugin {
 		const editor = this.editor;
 		const dropdownView = createDropdown( locale, SplitButtonView );
 		const mergeCommandName = 'mergeTableCells';
+		const splitButtonView = dropdownView.buttonView;
```

Comment:
This change is not necessary. It doesn't improve much: variable is used only once, there's only one member after a dot character so that's not a long access chain.